mlir/test/Dialect/Vector/vector-unroll-options.mlir

```diff
@@ -188,6 +188,15 @@ func.func @vector_fma(%a: vector<4x4xf32>, %b: vector<4x4xf32>, %c: vector<4x4xf
 //   CHECK-LABEL: func @vector_fma
 // CHECK-COUNT-4: vector.fma %{{.+}}, %{{.+}}, %{{.+}} : vector<2x2xf32>
 
+func.func @negative_vector_fma_3d(%a: vector<3x2x2xf32>) {
+  %0 = vector.fma %a, %a, %a : vector<3x2x2xf32>
+  return 
+}
```

Comment:
Right now it's not 100% obvious _why_ this case fails (but we know that it's the usage of 3d vectors that makes it unique). Please add a hint for our future-selves. Something like:
> // TODO: We should be able to unroll this like the example above - this will require extending `UnrollElementwisePattern`.

This way it is clear that the main reason for this to fail is simply missing functionality.

Reply:
How should the functionality be extended? By bumping the tile shape with prepended 1's? i.e. 2x2 to 1x2x2? Asking because this is similar to the PR https://github.com/llvm/llvm-project/pull/149239 (@CoTinker)